### ISSUE TITLE
[A11y][ARIA]NVDA is not reading the label name for the 'Select Scopes' checkboxes

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#8182]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1184483

## Browser Details:
Firefox Version: 79.0 (64 bit)
Edge Dev: Version 86.0.601.1 (Official build) dev (64-bit)
Screen Reader: NVDA, Narrator

## OS Details

Microsoft Windows 10 Enterprise  ​

Version 2004 (OS Build 19041.450)

## Repro Steps:

    Turn on NVDA.
    Hit the URL and login with appropriate credentials
    Tab till "Open Profile drop down" and press enter
    Tab till "API Keys" and Press enter
    Tab till "Create button" and press enter
    Tab till "Select scopes" section
    Check whether NVDA is reading the label name for the 'Select Scopes' 

## Actual Results:

NVDA is not reading the label name for the 'Select Scopes' Checkboxes

## Expected Results:
NVDA should read the label name for the "Select Scopes"  
## User Impact:
Users with visual disabilities may not understand the information clearly if screen reader doesn't read the label names

## Notes:
Check the aria-label or aria-labelled-by property on the scopes. Chances are, this isn't labelled correctly.
